lib/streamlit/web/server/websocket_headers.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from streamlit import runtime
+from typing import Optional, Dict
+
+from streamlit.runtime.scriptrunner import get_script_run_ctx
+from streamlit.web.server.browser_websocket_handler import BrowserWebSocketHandler
+
+
+def _get_websocket_headers() -> Optional[Dict[str, str]]:
```

Comment:
nit: is there a chance that returned value of  `_get_websocket_headers()` could be `str: List[str]`?

I looked to tornado `HttpHeaders` class and look's like the default behavior is comma joined list elements of same header multiple values, but this could be potentially different in other `session_client` implementations